snap/snapcraft.yaml

```diff
@@ -3,21 +3,21 @@ base: core22
 version: git
 summary: ROS 2 data exporter
 description: |
-  The ROS 2 data exporter is recording ROS 2 data
-  and exporting them on a server for later consultation
+  The ROS 2 data exporter records ROS 2 data as rosbags
+  and exports them to an S3 server for later use.
 
-  Add you private ssh-key and config file for the storage server in:
+  Add your config file for the storage server in:
 
-   /root/snap/ros2-exporter-agent/common/.ssh/
+  /root/snap/ros2-exporter-agent/current/.config/rclone/rclone.conf
 
   The `config` file should look like:
 
   ```
   Host storage-server
-      User <server_user>
-      HostName <server_hostname>
-      IdentityFile /root/snap/ros2-exporter-agent/common/.ssh/<my_private_key>
-      UserKnownHostsFile /root/snap/ros2-exporter-agent/common/.ssh/known_hosts
+    User <server_user>
```

Comment:
shouldn't this conf file example be changed?

Reply:
Yup, will do :+1: